pandas-stubs/_testing/__init__.pyi

```diff
@@ -32,14 +28,9 @@ def assert_almost_equal(
 ): ...
 def assert_dict_equal(left, right, compare_keys: bool = ...): ...
 def randbool(size=..., p: float = ...): ...
-
-RANDS_CHARS = ...
-RANDU_CHARS = ...
-
 def rands_array(nchars, size, dtype: str = ...): ...
 def randu_array(nchars, size, dtype: str = ...): ...
 def rands(nchars): ...
-def randu(nchars): ...
```

Comment:
I think there are only 4 public functions in `pandas.testing`:
```
    assert_extension_array_equal,
    assert_frame_equal,
    assert_index_equal,
    assert_series_equal,
```
So `pandas._testing` is private, and we just need to keep those 4 functions here and delete the rest.

Reply:
People might be using a few more, for example, https://www.programcreek.com/python/example/103667/pandas.util.testing.assert_almost_equal

Reply:
I'll remove all but the assert_* functions

Reply:
and `getSeriesData` as it is called in the tests

Reply:
I would be fine removing more in follow-up PRs.

The goal of this PR was initially to remove symbols that do not exist at runtime, and then it slowly grow when also removing obviously private symbols, and so on :)

Reply:
ok